src/app/_constants/api.ts

```diff
@@ -51,3 +51,9 @@ export const API_ROUTE_CLINICAL_VISIT = {
   GET_ALL: 'clinicalvisits/indexList',
   UPDATE: 'clinicalvisits/update',
 };
+
+export const API_ROUTE_VISIT = {
```

Comment:
can you please change this to plural like all the rest, to be consistant? also for the API_ROUTE_CLINICAL_VISIT